tools/cockpit.spec

```diff
@@ -175,67 +200,121 @@ install -p -m 644 cockpit.pp %{buildroot}%{_datadir}/selinux/targeted/
 %ifnarch x86_64
 rm -rf %{buildroot}/%{_datadir}/%{name}/docker
 %endif
+%if 0%{?suse_version}
+%suse_update_desktop_file -u -r -G 'Cockpit Server Manager' %{name} System Utility
+mv %{buildroot}%{_datadir}/doc/%{name} %{buildroot}%{_docdir}/%{name}/html
+%endif
 
 %files
+%if 0%{?suse_version}
+%defattr(-,root,root)
+%endif
 %{_docdir}/%{name}/AUTHORS
 %{_docdir}/%{name}/COPYING
 %{_docdir}/%{name}/README.md
+%if 0%{?suse_version}
+%dir %{_datadir}/%{name}
+%endif
 %{_datadir}/appdata
 %{_datadir}/applications
 %{_datadir}/pixmaps
 
 %files bridge
+%if 0%{?suse_version}
+%defattr(-,root,root)
+%endif
 %doc %{_mandir}/man1/cockpit-bridge.1.gz
 %{_bindir}/cockpit-bridge
 %attr(4755, -, -) %{_libexecdir}/cockpit-polkit
+%if 0%{?suse_version}
+%dir %{_libdir}/security
+%endif
 %{_libdir}/security/pam_reauthorize.so
 
 %files daemon
+%if 0%{?suse_version}
+%defattr(-,root,root)
+%endif
 %doc %{_mandir}/man8/cockpitd.8.gz
 %{_datadir}/dbus-1/services/com.redhat.Cockpit.service
 %{_libexecdir}/cockpitd
 
 %files doc
+%if 0%{?suse_version}
+%defattr(-,root,root)
+%endif
+%exclude %{_docdir}/%{name}/AUTHORS
+%exclude %{_docdir}/%{name}/COPYING
+%exclude %{_docdir}/%{name}/README.md
```

Comment:
Well, SUSE is a bit too strict about file conflicts. rpm allows conflicting files if they have the same size/checksum, SUSE doesn't.